### PR TITLE
Storybook > Footer

### PR DESCRIPTION
**Pull request by mackermedia**

### CODE REVIEW

File: components/ui/Footer.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+const Copyright = styled.div`
+  padding-top: 1.6rem;
+  color: ${theme.colors.white};
+`
+
+const Footer = ({ backgroundColor }) => (
+  <FooterContainer backgroundColor={backgroundColor}>
+    <FooterLinksContainer>
+      <FooterLink>
+        <Link href="#TODO" passHref>
```
Comment (mackermedia) [inline components/ui/Footer.js:45]: I don't think we know the final URL, correct?
Also, is it weird to link to the same website from ... the website?
Or is this some other website?

File: components/ui/Footer.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        <a
+          href="https://twitter.com/theindiedao"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <StyledLinkText>Twitter</StyledLinkText>
+        </a>
+      </FooterLink>
+      <FooterLink>
+        <Link href="#TODO" passHref>
```
Comment (mackermedia) [inline components/ui/Footer.js:68]: Is the IndieDAO Discord going to be open for all? 
I'm not sure how to find the correct URL for self-joiners...

**Comment by mckelvey** [comment]
[Storybook > Window Footer](https://trello.com/c/MkngUzKv/22-storybook-window-footer)

File: components/ui/Footer.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  background: ${props => props.backgroundColor}
+    ${props => props.theme.halftones.md};
+`
+
+const FooterLinksContainer = styled.div`
+  display: flex;
+  font-size: 2rem;
+  line-height: 2rem;
+
+  > :not([hidden]) ~ :not([hidden]) {
```
Comment (mckelvey) [inline components/ui/Footer.js:24]: I wouldn’t expect hidden to play a role here, how about: ` > *:nth-child(n+2) { border-left-.... }` or `:not(:first-child)` could also work.

File: components/ui/Footer.js (hunk 1 above)
Comment (mckelvey) [reply components/ui/Footer.js:45]: OK... this prompted a quick discussion... new links (updated in the Figma too):

1. Whitepaper
2. Twitter (https://twitter.com/lvlprotocol)
3. IndieDAO (https://twitter.com/theindiedao)
4. Gitbook (https://indiedao.gitbook.io/indiedao/)

File: components/ui/Footer.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    border-right-width: 0;
+    border-left-width: 0.1rem;
+    border-left-style: solid;
+    border-color: ${theme.colors.white};
+  }
+`
+
+const FooterLink = styled.div`
```
Comment (mckelvey) [inline components/ui/Footer.js:32]: Hmm.... (with the notes that these are all now external links) you could just `const A = styled.a` now...

File: components/ui/Footer.js (hunk 2 above)
Comment (mckelvey) [reply components/ui/Footer.js:68]: Did so... link changes above... sorry for the late notice on this, but your question was well thought!

**Review by mckelvey — CHANGES_REQUESTED** [review]
@mackermedia — This is a great push! Your question on discord prompted some changes, along with a few other minor details. Thanks!

**Review by royriver23 — COMMENTED** [review]
Other than @mckelvey comments, this looks awesomee!

File: components/ui/Footer.js (hunk 3 above)
Comment (mackermedia) [reply components/ui/Footer.js:24]: good call. I copied from Tailwind CSS's `divide-x` source!

**Comment by mackermedia** [comment]
@mckelvey most excellent feedback. Thanks!

**Pull request merged**